lib/valkyrie/persistence/solr/orm_converter.rb

```diff
@@ -32,32 +45,54 @@ def attributes
                            Valkyrie::Persistence::Attributes::OPTIMISTIC_LOCK => token)
     end
 
+    # Construct a Time object from the datestamp for the resource creation date indexed in Solr
+    # @return [Time]
     def created_at
       DateTime.parse(solr_document.fetch("created_at_dtsi").to_s).utc
     end
 
+    # Construct a Time object from the datestamp for the date of the last resource update indexed in Solr
+    # @return [Time]
     def updated_at
       DateTime.parse(solr_document["timestamp"] || solr_document.fetch("created_at_dtsi").to_s).utc
     end
 
+    # Construct the OptimisticLockToken object using the "_version_" field value in the Solr Document
+    # @see https://lucene.apache.org/solr/guide/7_4/updating-parts-of-documents.html#optimistic-concurrency
```

Comment:
Same as above, do we want to use `https://lucene.apache.org/solr/guide/updating-parts-of-documents.html#optimistic-concurrency` here?